src/core/evaluator.js

```diff
@@ -1459,17 +1459,17 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
               var isSameTextLine = !textState.font ? false :
                 ((textState.font.vertical ? args[0] : args[1]) === 0);
               advance = args[0] - args[1];
+              diff = (args[0] - textContentItem.lastAdvanceWidth) -
```

Comment:
It seems like the `advance` variable above this line is not used anymore, so we can probably remove it.